packages/pwa/app/constants.js

```diff
@@ -45,8 +45,11 @@ export const API_ERROR_MESSAGE = 'Something went wrong. Try again!'
 
 export const HOME_HREF = '/'
 
-// TODO: You can update these locales in 'pwa/package.json' file
-export const SUPPORTED_LOCALES = packageInfo.l10n.supportedLocales
-export const DEFAULT_LOCALE = packageInfo.l10n.defaultLocale
+export const SUPPORTED_LOCALES = localizationConfig.supportedLocales.map((locale) => locale.id)
+export const DEFAULT_LOCALE = localizationConfig.defaultLocale
+export const LOCALE_SELECTOR_MESSSAGES = localizationConfig.supportedLocales.reduce(
+    (messages, locale) => ({...messages, [locale.id]: locale.message}),
+    {}
+)
```

Comment:
This is where we can dice the localization config into smaller chunks... in ways that fit our business logic.

Reply:
I think we can put these three in the localization file instead of here as they don't look like constants to me. They are more like a derived data from the config file. 🤔 .

Reply:
🤔 If we consider the original config to be constants, then I'd say any derived subset of it would also be constants 😄